pkg/cloud/models/models.go

```diff
@@ -181,6 +181,8 @@ type CustomPricing struct {
 	AzureStorageContainer        string `json:"azureStorageContainer"`
 	AzureContainerPath           string `json:"azureContainerPath"`
 	AzureCloud                   string `json:"azureCloud"`
+	AzureResourceGroupName       string `json:"azureResourceGroupName"`
+	AzureClusterName             string `json:"azureClusterName"`
```

Comment:
I would prefer to avoid routing additional values through through the custom pricing struct, and this does not seem necessary as the env variables can be checked at the beginning of the price checking process without saving them here.

Reply:
since they are Azure specific values I would prefer them to be properties of the Azure struct.

Reply:
Thanks for the suggestion. Fixed it.

Reply:
Thanks for the review @Sean-Holcomb . Fixed it. Please review.